component/siddhi-io-nats.iml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Exclude .iml files

Reply:
Resolved with the commit 508fb521854cc962d479d5cb9d2099973b53d4bb